docs/python-api/introduction.rst

```diff
@@ -1,35 +1,35 @@
-Introduction
-============
-|neptune-client-github| is open source Python library that allows you to integrate your Python scripts with Neptune. Neptune client supports handful of use cases:
+About neptune-client
+--------------------
 
-* creating and tracking experiments
-* managing running experiment
-* querying experiments and projects (search/download)
+|neptune-client-github|
+is an open source Python library that lets you integrate your Python scripts with Neptune so that you can more easily track and organize your experiments in the rich Neptune `dashboard <https://ui.neptune.ai/shared/onboarding/experiments>`_.
```

Comment:
Let's put this link: https://ui.neptune.ai/o/USERNAME/org/example-project/experiments?viewId=d7f80ebe-5bfe-4d12-97c1-2b1e6184a2ed